src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -203,6 +207,17 @@ IntRect RenderTarget::getViewport(const View& view) const
 }
 
 
+////////////////////////////////////////////////////////////
+IntRect RenderTarget::getScissor(const View& view) const
+{
+    const auto [width, height] = Vector2f(getSize());
+    const FloatRect& scissor   = view.getScissor();
+
+    return IntRect(Rect<long>({std::lround(width * scissor.left), std::lround(height * scissor.top)},
+                              {std::lround(width * scissor.width), std::lround(height * scissor.height)}));
```

Comment:
Any reason to prefer `std::lround` over `std::round`?

Reply:
`std::round` returns a floating point type and since we need need an integral type, it's easier to use `std::lround` which returns a `long`.